### PR TITLE
Top sold product api

### PR DESCRIPTION
**Pull request by rochiblau**

### CODE REVIEW

**Comment by rochiblau** [comment]
which exseptions should I do?

File: src/main/java/com/sapred/ordermanagerred/service/GraphService.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    @Autowired
+    private JwtToken jwtToken;
+
+    public List<MonthProductCountDto> getTopProduct(String token, int rangeOfMonths) {
+        String companyId = jwtToken.getCompanyIdFromToken(token);
+        LocalDate dateForLastThreeMonth = LocalDate.now().minusMonths(rangeOfMonths);
+        Aggregation aggregation = Aggregation.newAggregation(
+
+                match(Criteria.where("companyId.$id").is(companyId)
+                        .and("orderStatus").is("DONE")
```
Comment (moriyahazan) [inline src/main/java/com/sapred/ordermanagerred/service/GraphService.java:32]: Please use enum constant for order status and not string "Done"

File: src/main/java/com/sapred/ordermanagerred/dto/ProductCountDto.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@SuperBuilder(toBuilder = true)
+public class ProductCountDto {
+    private String productId;
```
Comment (moriyahazan) [inline src/main/java/com/sapred/ordermanagerred/dto/ProductCountDto.java:13]: Please return also product name since name is displayed in screen

**Review by moriyahazan — APPROVED** [review]
Very nice 
please see my small comments, fix, merge from main and close

**Pull request closed without merging**